static/src/stylesheets/module/survey/_weekend-reading.scss

```diff
@@ -0,0 +1,177 @@
+.email-signup-main {
+    background-color: $news-main-1;
+}
+
+.email-signup-main:before {
+    height: 26px;
+    content: '';
+    background-color: $news-main-1;
+    display: block;
+}
+
+.email-signup {
+    background-color: $news-main-1;
+    min-width: 16em;
+}
+
+.content {
```

Comment:
I'm guessing the survey CSS is only included on the survey pages so we don't need to worry about styles getting overridden elsewhere in the site?